etc/spack/defaults/config.yaml

```diff
@@ -81,6 +81,11 @@ config:
   misc_cache: ~/.spack/cache
 
 
+  # The binary cache index tracks what specs are available on (usually remote)
+  # binary caches. Default location is $spack/opt/spack/indices
+  # binary_index_root: ~/.spack/indices
```

Comment:
```suggestion
  # binary caches. In case $spack is read-only, set to a writeable directory, like   ~/.spack/indices. 
  binary_index_root: $spack/opt/spack/indices
```

Convention seems to be to explicitly set the default here (this makes the default value that is set in `lib/spack/spack/binary_distribution.py` ineffective, maybe remove there?)